styles/components/Navbar.scss

```diff
@@ -3,199 +3,240 @@
-      padding: 0;
-      text-align: center;
-    }
+      margin: 0 20px;
+      overflow: hidden;
+      position: relative;
+      width: 50px;
+      z-index: 12;
 
-    ul.nav-items li {
-      display: block;
-      font-size: 1.8em;
-      font-weight: 600;
-      margin: 40px auto;
-      opacity: 0;
-      padding: 0;
-      text-transform: lowercase;
-      transform: translate3d(0, -50%, 0);
-      transition: all 1.25s cubic-bezier(0.05, 1.04, 0.72, 0.98);
-    }
+      .bar {
+        background-color: $font-secondary-color;
+        border-radius: 04px;
+        height: 03px;
+        position: relative;
+        transition: all 0.5s ease-in-out;
+        width: 50%;
+
+        &::before,
+        &::after {
+          background-color: $font-secondary-color;
+          border-radius: 04px;
+          content: '';
+          height: 03px;
+          left: 0;
+          position: absolute;
+          transition: all 0.5s ease-in-out;
+          width: 100%;
+        }
+
+        &::before {
+          transform: translateY(8px);
+        }
+
+        &::after {
+          transform: translateY(-8px);
+        }
+      }
 
-    ul.nav-items li:hover {
-      color: #999;
-      cursor: pointer;
+      &.active {
+        .bar {
+          transform: translateX(-50px);
+        }
 
+        .bar::before {
+          transform: rotate(-45deg) translate(35px, 35px);
+        }
 
-    }
+        .bar::after {
+          transform: rotate(45deg) translate(35px, -35px);
+        }
+      }
 
-    @media (max-height: 800px) {
-      ul.nav-items li {
-        margin: 20px auto;
+      &:hover {
+        .bar,
+        .bar::before,
+        .bar::after {
+          background-color: $acm-cobalt;
+        }
       }
     }
-    @media (max-height: 600px) {
-      ul.nav-items li {
-        margin: 5px auto;
+
+    .nav-items {
+      align-items: center;
+      display: flex; // will become none at smaller screen size
+      list-style: none;
+      margin: 0;
+      padding: 0;
+
+      li {
+        line-height: 40px;
+        padding: 0 20px;
+
+        button {
+          background-color: transparent;
+          border: 0;
+          color: inherit;
+          font-size: 0.95em;
+          font-weight: 600;
+          text-transform: lowercase;
+          transition: 0.2s;
+
+          &.active {
+            color: $acm-cobalt;
+          }
+          
+          &:hover {
+            color: $acm-cobalt;
+            cursor: pointer;
+          }
+        }
+
+        // prevents committee text fade on li hover
+        &:hover {
+          color: $font-primary-color;
+        }
       }
-    }
-    @media (max-height: 400px) {
-      ul.nav-items li {
-        margin: 1px auto;
+
+      #committee-mobile-nav-item {
+        display: none;
+        height: max-content;
+        text-align: center;
+        width: 60%;
+
+        li {
+          padding: 0;
+        }
+
+        // committee color applied on hover and active
```

Comment:
Two quick things here:

* this is quite a bit of duplication! I'm fine with this (since it's hard), but you may want to look into [looping with `@for` in SASS](https://sass-lang.com/documentation/at-rules/control/for). we don't have to do this if it's confusing to implement though
* instead of using these hex colors, could we use the [SASS variables for each committee](https://github.com/uclaacm/website/blob/main/styles/variables.scss)?